scss/_patterns_card.scss

```diff
@@ -31,15 +9,28 @@
     overflow: auto; // prevent overflow of children margins
     padding: $spv-intra--scaleable;
 
-    .p-card__content {
-      margin-top: - $sp-unit * 2;
+    img {
```

Comment:
All patterns should only target classes. This allowed to pattern to be weekend agnostic.

Reply:
the trouble with this is, there already is an image with no class inside the header card pattern. Moving this rule to a class would require editing all instances of markup on all sites. It is a simple search and replace, so not too difficult, but still - is that ok?

Reply:
ok

Reply:
Can we create a class for it too and update the example to use the class so we can phase out this issue.

Reply:
I've added a class .p-card__thumbnail that sets max-height to the current 2rem. @kwm14 Would be good to audit these and see if we can reduce the number of different sizes. I'm putting them in a sass map, so far I've got:

  .75rem - accordion icons
  1rem - media object meta icons, notification icons
  2rem - card header thumbnail image
  2.5rem - social icons 
  3rem - matrix
  5rem - media object
  6rem - media object large